### PR TITLE
[C-2337] Remove reachability mobile web

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Remove reachability on mobile web & also use cloudflare for /204 instead of GA.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Need to further test on stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-remove-reachability-mobile-web

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-remove-reachability-mobile-web

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-remove-reachability-mobile-web

File: packages/web/src/common/store/profile/sagas.js [hunk 1]
```diff
@@ -124,6 +124,7 @@ export function* fetchOpenSeaAssets(user) {
   const { wallets } = yield apiClient.getAssociatedWallets({
     userID: user.user_id
   })
+  console.log('raymont', wallets)
```
Comment (sddioulde) [inline packages/web/src/common/store/profile/sagas.js:127]: log

**Review by sddioulde — APPROVED** [review]
besides log, looks good to me

**Review by amendelsohn — APPROVED** [review]

**Pull request merged**